docs/installation/sources/sources_windows.rst

```diff
@@ -0,0 +1,241 @@
+.. _windows_sources:
+
+Windows installation from sources
+=================================
+
+To install *eProsima Fast DDS* from sources, you first need to meet the required dependencies
+(see requirements_windows_sources_)
+and then choose whether to follow either the colcon_ (see colcon colcon_installation_windows_) or the CMake_
+(see cmake_installation_windows_) installation instructions.
+
+.. _requirements_windows_sources:
+
+Requirements
+------------
+
+*eProsima Fast RTPS* requires the following dependencies when building from sources in a Windows environment.
+
+Visual Studio
+^^^^^^^^^^^^^
+
+To install *eProsima Fast DDS*, you need to have `Visual Studio <https://visualstudio.microsoft.com/>`_ installed in
+your system. Make sure you check the :code:`Desktop development with C++` option during the installation process.
+If you have Visual Studio but you haven’t the Visual C++ Redistributable packages installed,
+open Visual Studio and go to :code:`Tools` -> :code:`Get Tools and Features` and in the :code:`Workloads` tab enable
+:code:`Desktop development with C++`. Click :code:`Modify` at the bottom right when you're done.
```

Comment:
You can just refer to the first explanation of this

Reply:
That in the binaries?
I don't like this solution. I think that the purpose of these pages is that anyone should just choose the options combination he/she wants (OS + sources/binaries) and stick to that,m without having to go navigating through the pages.